lib/ReportDataCollector.php

```diff
@@ -117,6 +123,9 @@ public function __construct(
 		$this->systemConfig = $systemConfig;
 		$this->apps = \OC_App::listAllApps();
 		$this->appConfig = $appConfig;
+
+		$event = new ConfigReportEvent();
+		$this->appConfigData = \OC::$server->getEventDispatcher()->dispatch('OCA\ConfigReport::loadData', $event)->getConfigReportData();
```

Comment:
At first I thought it would be good to let plugins inject values / details in existing arrays.
But this way is fine too.